src/main/kotlin/org/opensearch/commons/alerting/model/ClusterMetricsInput.kt

```diff
@@ -74,6 +75,7 @@ data class ClusterMetricsInput(
             .field(PATH_FIELD, path)
             .field(PATH_PARAMS_FIELD, pathParams)
             .field(URL_FIELD, url)
+            .field(CLUSTERS_FIELD, clusters)
```

Comment:
same.. update parsing tests for clusterMetricsInput model or plz add a test if non-existent

Reply:
Revised.